lib/shoryuken/queue.rb

```
@@ -120,6 +124,24 @@ def fifo?
+    # duplicate message_deduplication_id) rather than raising, so without this
+    # they are silently swallowed - notably by ShoryukenAdapter#enqueue_all,
+    # which reads only response.successful. Mirrors delete_messages.
+    #
+    # @param response [Aws::SQS::Types::SendMessageBatchResult] the batch result
+    # @return [void]
+    def log_failed_sends(response)
+      return unless response.respond_to?(:failed)
+
+      Array(response.failed).each do |failure|
+        logger.error do
+          "Could not send #{failure.id}, code: '#{failure.code}', message: '#{failure.message}', sender_fault: #{failure.sender_fault}"
```

RuboCop (1.88.0)

[convention] Line is too long. [135/125]

(Layout/LineLength)